wordpress-mobile/WordPress-iOS
# Better offline messages in People list

Implements part of  #5759 . Better error messages in People when there is no network connection

To test:
1. Enter flight mode
2. Navigate to any site > People

![Content cached](https://user-images.githubusercontent.com/2722505/36956231-0e969fca-1fe2-11e8-8617-59d61ff4f2cb.PNG)
![No content cached](https://user-images.githubusercontent.com/2722505/36956239-1b99fbd6-1fe2-11e8-9f47-484d7c27c425.PNG)
![No content cached, alert dismissed](https://user-images.githubusercontent.com/2722505/36956243-1e3a3540-1fe2-11e8-8da7-dfcd511ade93.PNG)

Same as in the previous PR, should we aim to refresh the content automatically when the network connection is back up?
